lib/yocaml/effect.ml

```diff
@@ -2,12 +2,23 @@ open Util
 
 type (_, 'a) effects =
   | File_exists : Filepath.t -> (< file_exists : unit ; .. >, bool) effects
+  | Target_exists :
+      Filepath.t
+      -> (< target_exists : unit ; .. >, bool) effects
   | Get_modification_time :
       Filepath.t
       -> (< get_modification_time : unit ; .. >, int Try.t) effects
+  | Target_modification_time :
+      Filepath.t
+      -> (< target_modification_time : unit ; .. >, int Try.t) effects
   | Read_file :
       Filepath.t
       -> (< read_file : unit ; .. >, string Try.t) effects
+  | Content_changes :
+      (Filepath.t * string)
+      -> ( < content_changes : unit ; .. >
+         , (string, unit) Either.t Try.t )
+         effects
```

Comment:
I think that the effect should be: 

```ocaml
| Content_changes : (Filepath.t * string) -> ( < content_changes : unit ; .. >,  bool Try.t ) effects
```

And changing the function `content_changes` to be iso with the runtime

Reply:
I disagree :)